packages/esm-service-queues-app/src/queue-patient-linelists/queue-linelist-filter.test.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import userEvent from '@testing-library/user-event';
 import { render, screen } from '@testing-library/react';
-import { useLayoutType, useVisitTypes } from '@openmrs/esm-framework';
+import { OpenmrsDatePicker, useLayoutType, useVisitTypes } from '@openmrs/esm-framework';
 import { mockVisitTypes } from '__mocks__';
 import QueueLinelistFilter from './queue-linelist-filter.workspace';
+import dayjs from 'dayjs';
 
 const mockUseLayoutType = jest.mocked(useLayoutType);
 const mockUseVisitTypes = jest.mocked(useVisitTypes);
+const mockOpenmrsDatePicker = jest.mocked(OpenmrsDatePicker);
```

Comment:
Do things break if we _don't_ mock this? If they do, we should look into fixing that. I don't think mocking components (as opposed to data fetching hooks) is a good idea.